test/integration/lerna-bootstrap.test.js

```diff
@@ -21,6 +21,10 @@ describe("lerna bootstrap", () => {
           .then((result) => {
             expect(result.stdout).toMatchSnapshot("stdout: simple");
           })
+          .then(() => execa(LERNA_BIN, ["bootstrap", "--ignore", "@integration/package-1"], { cwd }))
```

Comment:
This needs to run in a seperate `test.concurrent()` block. It's not isolated enough, here. (which is testing the "defaults")